test/utils/CurrencySettlerTestUtil.sol

```diff
@@ -9,7 +9,9 @@ import {IPoolManager} from "../../src/interfaces/IPoolManager.sol";
 /// To settle a positive delta (a credit to the user), a user may take or mint.
 /// To settle a negative delta (a debt on the user), a user make transfer or burn to pay off a debt.
 /// @dev Note that sync() is called before any erc-20 transfer in `settle`.
-library CurrencySettler {
+/// @dev THIS CONTRACT IS NOT SECURE, IT IS FOR TESTING PURPOSES, DO NOT USE IT IN PROD
```

Comment:
I feel like this warning is a bit overkill? It just doesn't use safe transfer from and we don't do this in a lot of our test contracts which should never be used in prod

Reply:
I think just renaming it and putting it in the test/utils file is sufficient